Software/SoftwareBukkitSpigotETC.md

```diff
@@ -8,16 +8,15 @@ Join our [discord](https://discord.gg/8nzHYhVUQS) or use the issues.<br><br>
-| [MinecraftMist](https://github.com/MinecraftMist/Mist) | Forge | 1.16.5 | Hybrid server software for Minecraft, allowing Forge mods and Bukkit plugins to run together. Rogue fork of Mohist. |
-| [MohistMC](https://mohistmc.com) | Forge | 1.7.10, 1.12.2 & 1.16.5 | Minecraft Forge Server Software Implementing Paper/Spigot/Bukkit API. |
-| [ThermosX](https://github.com/DireClanMC/ThermosX) | Forge | 1.7.10 | ThermosX is a fork of KCauldron and Thermos, a CraftBukkit-Forge server for Minecraft 1.7.10. After periods of inactivity on KCauldron's (and Thermos) GitLab concerning major issues, ThermosX was created to allow active members of the Minecraft coding community to optimize it and provide fixes in a timely manner. |
-| [Thermos](https://cyberdynecc.github.io/Thermos/install) | Forge | 1.7.10 | Thermos is a fork of KCauldron, a craftbukkit forge server for Minecraft 1.7.10. |
+| Name | License | Supported Loaders | Supported Versions | Description |
+| --- | :---: | :---: | :---: | :---: |
+| [Arclight](https://github.com/IzzelAliz/Arclight) | Free - Open Source | Forge | 1.14.x - 1.18.x | A Bukkit server implementation utilizing Mixin. |
+| [Cardboard](https://www.curseforge.com/minecraft/mc-mods/cardboard) | Free - Open Source | Fabric | 1.16.3 - 1.18 | Cardboard is acutally a mod but i wanted a option for Fabric here also. Cardboard is a Fabric mod that adds support for the popular Bukkit/Spigot modding API. This mod lets you use plugins that are made for Bukkit and it's derivatives (Spigot & Paper) with Fabric. |
+| [Crucible](https://github.com/CrucibleMC/Crucible) | Free - Open Source | Forge | 1.7.10 | Crucible is a continuation of Thermos as it has been discontinued. |
+| [Glowstone](https://github.com/GlowstoneMC/Glowstone) | Free - Open Source | Vanilla | 1.12 | Glowstone is a lightweight, from scratch, open source Minecraft server written in Java that supports plugins written for the Bukkit API and its major forks, Spigot and Paper. | 
+| [Magma](https://magmafoundation.org) | Free - Open Source | Forge | 1.12.2 & 1.16.5 | Minecraft Forge Server Software Implementing Paper/Spigot/Bukkit API. |
+| [MinecraftMist](https://github.com/MinecraftMist/Mist) | Free - Open Source | Forge | 1.16.5 | Hybrid server software for Minecraft, allowing Forge mods and Bukkit plugins to run together. Rogue fork of Mohist. |
+| [MohistMC](https://mohistmc.com) | Free - Open Source | Forge | 1.7.10, 1.12.2 & 1.16.5 | Minecraft Forge Server Software Implementing Paper/Spigot/Bukkit API. |
+| [ThermosX](https://github.com/DireClanMC/ThermosX) | Free - Open Source | Forge | 1.7.10 | ThermosX is a fork of KCauldron and Thermos, a CraftBukkit-Forge server for Minecraft 1.7.10. After periods of inactivity on KCauldron's (and Thermos) GitLab concerning major issues, ThermosX was created to allow active members of the Minecraft coding community to optimize it and provide fixes in a timely manner. |
```

Comment:
oh yeah i forgot to mention since Thermos seemed to be completely superceded by ThermosX, i removed it's entry